examples/model_loader.py

```diff
@@ -0,0 +1,94 @@
+from yggdrasil.examples import yamls as ex_yamls
+import yaml
+import jupyterlab_nodeeditor as jlne
+
+def dict_conversion(model_file):
```

Comment:
Is this a public-facing function?  If so, it needs a docstring.  If not, it should be prefixed with an underscore.

Reply:
Added underscore for clarity